config/superblock-order.ts

```diff
@@ -0,0 +1,339 @@
+import { Languages } from './i18n/all-langs';
+import { SuperBlocks } from './certification-settings';
+
+enum SuperBlockStates {
+  Audited = 'audited',
+  NotAudited = 'notAudited',
+  Upcoming = 'upcoming',
+  Legacy = 'legacy'
+}
+
+enum CurriculumMaps {
+  Landing = 'landing',
+  Learn = 'learn'
+}
+
+/* The order of superblocks in the arrays below are how they appear on the maps
+ *
+ * The 'Landing' map array should contain exactly one superblock for each
+ * current/non-legacy certification, and only one superblock of each type -
+ * e.g. only one RWD superblock/button
+ *
+ * The 'Learn' map arrays should contain all available superblocks, sorted into
+ * their various states.
+ */
+
+export const superBlockOrder = {
+  [Languages.English]: {
+    [CurriculumMaps.Landing]: [
+      SuperBlocks.RespWebDesignNew,
+      SuperBlocks.JsAlgoDataStruct,
+      SuperBlocks.FrontEndDevLibs,
+      SuperBlocks.DataVis,
+      SuperBlocks.RelationalDb,
+      SuperBlocks.BackEndDevApis,
+      SuperBlocks.QualityAssurance,
+      SuperBlocks.SciCompPy,
+      SuperBlocks.DataAnalysisPy,
+      SuperBlocks.InfoSec,
+      SuperBlocks.MachineLearningPy
+    ],
+    [CurriculumMaps.Learn]: {
+      [SuperBlockStates.Audited]: [
+        SuperBlocks.RespWebDesignNew,
+        SuperBlocks.JsAlgoDataStruct,
+        SuperBlocks.FrontEndDevLibs,
+        SuperBlocks.DataVis,
+        SuperBlocks.RelationalDb,
+        SuperBlocks.BackEndDevApis,
+        SuperBlocks.QualityAssurance,
+        SuperBlocks.SciCompPy,
+        SuperBlocks.DataAnalysisPy,
+        SuperBlocks.InfoSec,
+        SuperBlocks.MachineLearningPy,
+        SuperBlocks.CodingInterviewPrep
+      ],
+      [SuperBlockStates.NotAudited]: [],
+      [SuperBlockStates.Upcoming]: [SuperBlocks.JsAlgoDataStructNew],
+      [SuperBlockStates.Legacy]: [SuperBlocks.RespWebDesign]
```

Comment:
```suggestion
  [Languages.English]: {
    [CurriculumMaps.Landing]: [
      SuperBlocks.RespWebDesignNew,
      SuperBlocks.JsAlgoDataStruct,
      SuperBlocks.FrontEndDevLibs,
      SuperBlocks.DataVis,
      SuperBlocks.RelationalDb,
      SuperBlocks.BackEndDevApis,
      SuperBlocks.QualityAssurance,
      SuperBlocks.SciCompPy,
      SuperBlocks.DataAnalysisPy,
      SuperBlocks.InfoSec,
      SuperBlocks.MachineLearningPy
    ],
    [CurriculumMaps.Learn]: [
        SuperBlocks.RespWebDesignNew,
        SuperBlocks.JsAlgoDataStruct,
        Headers.NotAudited,
        SuperBlocks.JsAlgoDataStructNew,
        SuperBlocks.FrontEndDevLibs,
        SuperBlocks.DataVis,
        SuperBlocks.RelationalDb,
        SuperBlocks.BackEndDevApis,
        SuperBlocks.QualityAssurance,
        SuperBlocks.SciCompPy,
        SuperBlocks.DataAnalysisPy,
        SuperBlocks.InfoSec,
        SuperBlocks.MachineLearningPy,
        SuperBlocks.CodingInterviewPrep,
        Headers.Legacy,
        SuperBlocks.RespWebDesign
      ]
```

Discussing w/ tom on meeting:

What if we had headers in the order - we could filter those out when we need to look at the superblocks specifically, and use them in the rendering logic to add titles so we can stick an upcoming block in the middle of the map?